app/src/components/Tabs/my-inventories-tab.tsx

```diff
@@ -183,7 +183,12 @@ const MyInventoriesTab: FC<MyInventoriesTabProps> = ({
                       <Box display="flex" gap="8px" alignItems="center">
                         <CircleFlag
                           countryCode={
-                            city.countryLocode?.substring(0, 2)?.toLowerCase() || ""
+                            city.countryLocode
+                              ?.substring(0, 2)
+                              ?.toLowerCase() ||
+                            city.locode?.substring(0, 2)?.toLowerCase() ||
+                            city.regionLocode?.substring(0, 2)?.toLowerCase() ||
+                            ""
```

Comment:
This is starting to get a bit long.
Can you extract this to a const definition at the start of the function so this can just be `countryCode={countryCode}` in the markup part at the end of the component?